lib/fetch/util.js

```diff
@@ -253,20 +253,20 @@ function coarsenedSharedCurrentTime (crossOriginIsolatedCapability) {
   return performance.now()
 }
 
-function makeTimingInfo (init) {
+// https://fetch.spec.whatwg.org/#create-an-opaque-timing-info
+function createOpaqueTimingInfo (timingInfo) {
   return {
-    startTime: 0,
+    startTime: timingInfo.startTime ?? 0,
     redirectStartTime: 0,
     redirectEndTime: 0,
-    postRedirectStartTime: 0,
+    postRedirectStartTime: timingInfo.startTime ?? 0,
     finalServiceWorkerStartTime: 0,
     finalNetworkResponseStartTime: 0,
     finalNetworkRequestStartTime: 0,
     endTime: 0,
     encodedBodySize: 0,
     decodedBodySize: 0,
-    finalConnectionTimingInfo: null,
-    ...init
```

Comment:
I removed this because the spec states to "return a new fetch timing info whose start time and post-redirect start time are timingInfo’s start time.". Since only startTime and postRedirectStartTime can be changed it seems more explicit to show that their values are the only ones being updated.